pkg/infra/pulumi_aws/iac/eks.ts

```diff
@@ -626,64 +654,70 @@ export class Eks {
         }
 
         if (image) {
-            const execUnitEdgeId = `${execUnit}_exec_unit`
-            let needsProxy = false
-            let needsGatewayLink = false
-
-            lib.topology.topologyIconData.forEach((resource) => {
-                if (resource.kind === Resource.gateway) {
-                    lib.topology.topologyEdgeData.forEach((edge) => {
-                        if (edge.source == resource.id && edge.target === execUnitEdgeId) {
-                            // We know that this exec unit is exposed and must create the necessary resources
-                            needsGatewayLink = true
-                        }
-                    })
+            console.log(needsLoadBalancer)
```

Comment:
is this old debug?

Reply:
topology is kind of blocked, i cut klotho #111 so we can discuss and fix it